.github/workflows/build-deploy-pudl.yml

```diff
@@ -0,0 +1,109 @@
+name: build-deploy-pudl
+on:
+  push:
+    branches:
+      - gce-deploy # for testing purposes
+    tags:
+      - "v**"
+  schedule:
+    - cron: "0 6 * * 1-5" # Weekdays at midnight on MST
+
+env:
+  PROJECT_ID: ${{ secrets.GCE_PROJECT }}
+  CHECKOUT_BRANCH: ${{ github.ref_name }}
+  GCE_INSTANCE: pudl-deployment-tag
```

Comment:
Is the instance always called `pudl-deployment-tag` even when it's running on a schedule? Oh wait no I see this is overridden below. A comment here indicating that would be helpful.

Reply:
Done.